wiki/Gameplay/Game_modifier/Hard_Rock/id.md

```diff
@@ -23,48 +21,52 @@ outdated_since: bb68816d3f5cdcce9eaa78788e9fc7b671e1dc93
+Mod **Hard Rock** adalah sebuah [game modifier](/wiki/Game_modifier) yang bertujuan untuk meningkatkan tingkat kesulitan pada sebagian besar aspek dari sebuah [beatmap](/wiki/Beatmap). Modifier ini meningkatkan [circle size (CS)](/wiki/Beatmap_Editor/Song_Setup#circle-size) sebesar 30%, [approach rate (AR)](/wiki/Beatmapping/Approach_rate) sebesar 40%, [health drain (HP)](/wiki/Beatmapping/Health_drain) sebesar 40% dan [overall difficulty (OD)](/wiki/Beatmap_Editor/Song_Setup#overall-difficulty) sebesar 40%, meskipun detailnya bervariasi untuk setiap mode permainan.
 
 ### osu!
 
-Pada [osu!](/wiki/Game_mode/osu!), (bersama dengan pengaturan kesulitan yang telah ditingkatkan) mod Hard Rock membalikkan posisi sumbu-X sebuah beatmap (ditunjukkan dibawah). Hal ini kadang-kadang bisa menyebabkan [pola](/wiki/Beatmap/Pattern) tertentu terasa tidak benar, yang bisa meningkatkan kesulitan.
+Pada [osu!](/wiki/Game_mode/osu!), (bersamaan dengan peningkatan pengaturan kesulitan) mod Hard Rock membalikkan beatmap pada sumbu X-nya (seperti yang ditunjukkan di bawah).Hal ini terkadang dapat membuat [pola](/wiki/Beatmap/Pattern) tertentu terasa tidak alami, yang dapat meningkatkan kesulitan.
 
-Sebagai tambahan, jika pemain telah menghafal atau sudah terbiasa dengan gerakan dan pola sebuah beatmap, beralih ke mod Hard Rock bisa menghilangkan penghafalan tersebut dan menyebabkan semacam sebuah kesulitan palsu.
+Selain itu, jika pemain telah menghafal atau terbiasa dengan gerakan dan pola beatmap, beralih ke mod Hard Rock dapat menghilangkan hafalan tersebut dan menyebabkan semacam kesulitan buatan.
 
-![Tampilan permainan HR](img/HR-comparison-osu.jpg "Perbandingan dari sebuah map tanpa mod terpasang (kiri) dengan map yang sama dengan mod Hard Rock dipasang (kanan) pada mode osu!")
+![Tampilan permainan HR](img/HR-comparison-osu.jpg "Perbandingan permainan biasa (kiri) dengan permainan dengan mod Hard Rock diaktifkan (kanan) pada mode osu!")
 
 ### osu!taiko
 
-Pada [osu!taiko](/wiki/Game_mode/osu!taiko), mod Hard Rock meningkatkan [slider velocity](/wiki/Hit_object/Slider_velocity) dan juga [overall difficulty (OD)](/wiki/Beatmapping/Overall_difficulty). Mod ini adalah mod yang biasa digunakan secara luas pada osu!taiko karena mod ini adalah satu-satunya mod peningkat kesulitan yang tidak membutuhkan penghafalan atau mengetuk dengan sangat cepat pada tingkat kesulitan yang lebih tinggi.
+Pada [osu!taiko](/wiki/Game_mode/osu!taiko), mod Hard Rock meningkatkan [slider velocity](/wiki/Hit_object/Slider_velocity) serta [overall difficulty (OD)](/wiki/Beatmapping/Overall_difficulty).Mod ini dianggap secara luas sebagai mod yang paling umum digunakan pada osu!taiko karena mod ini adalah satu-satunya mod peningkat kesulitan yang tidak membutuhkan penghafalan atau mengetuk dengan sangat cepat pada tingkat kesulitan yang lebih tinggi.
```

Comment:
```suggestion
Pada [osu!taiko](/wiki/Game_mode/osu!taiko), mod Hard Rock akan meningkatkan *[slider velocity](/wiki/Hit_object/Slider_velocity) dasar serta [overall difficulty (OD)](/wiki/Beatmapping/Overall_difficulty) yang aktif. Secara umum, mod ini merupakan mod yang paling sering digunakan pada osu!taiko karena mod ini adalah satu-satunya mod peningkat kesulitan yang tidak membutuhkan pemain untuk menghafal ataupun mengenai objek yang ada dengan sangat cepat.
```

Reply:
slider velocity is italicised but not overall difficulty?

Reply:
hmm you can un-italicize both if you're looking for consistency i think